moped-editor/src/views/projects/projectView/ProjectComponents/CreateComponentDrawTools.js

```diff
@@ -1,23 +1,23 @@
 import React, { useRef } from "react";
 import ComponentsDrawControl from "src/components/Maps/ComponentsDrawControl";
 import { makeDrawnFeature } from "./utils/features";
+import mapboxDrawStylesOverrides from "src/styles/mapboxDrawStylesOverrides";
 
 /**
- * Renders project component draw tools
+ * Renders project component create draw tools
  * @param {Function} createDispatch - dispatch to call create actions
  * @param {String} linkMode - tracks if we are editing "lines" or "points"
  * @param {Function} setCursor - function to update the cursor type
  * @param {Function} setIsDrawing - function to update if we are drawing or not
  * @returns {JSX.Element}
  */
-const ComponentDrawTools = ({
+const CreateComponentDrawTools = ({
   createDispatch,
   linkMode,
   setCursor,
   setIsDrawing,
 }) => {
   const drawControlsRef = useRef();
-  const shouldShowDrawControls = linkMode === "points" || linkMode === "lines";
```

Comment:
Moving this outside of this component so we can show/hide the create or edit draw tools inside of main map component.